### PR TITLE
Revert to single model

### LINKED ISSUE #45: Implement separate userTypes on Dashboard

### PR DESCRIPTION
**Pull request by maripot**
Closes #45

### CODE REVIEW

File: app/models/ability.rb [hunk 1]
```diff
@@ -4,8 +4,10 @@ class Ability
   def initialize(user)
     user ||= User.new # guest user (not logged in)
     if user.has_role?(:admin)
-      can :manage, :all
+      can [:edit, :delete, :new, :create, :update, :destroy], [Course, Offering, Semester] 
+      can :read, :all
     else
+   # can [:edit, :delete, :new, :create, :update, :destroy], [Enrollment]
```
Comment (alexthemitchell) [inline app/models/ability.rb]: Please remove if unneeded.

Comment (alexthemitchell) [inline app/models/ability.rb]: Make a ticket to re-add it if that's relevant.

File: app/controllers/users/registrations_controller.rb [hunk 2]
```diff
@@ -8,9 +8,15 @@ class Users::RegistrationsController < Devise::RegistrationsController
   # end
 
   # POST /resource
-  # def create
-  #   super
-  # end
+  def create
+    super
+    if resource.save
+      if resource.email == "sams@sam.r"
```
Comment (alexthemitchell) [inline app/controllers/users/registrations_controller.rb]: Do we want to use Sam's real email here?

Comment (maripot) [inline app/controllers/users/registrations_controller.rb]: you're right, might as well put that in right now -

**Pull request merged**